var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -244,38 +244,43 @@ def install(self, spec, prefix):
             print('========= Step-40 Trilinos ==========')
             print('=====================================')
             # change Linear Algebra to Trilinos
-            filter_file(r'(\/\/ #define FORCE_USE_OF_TRILINOS.*)',
-                        ('#define FORCE_USE_OF_TRILINOS'), 'step-40.cc')
+            if spec.satisfies('@8.4.0:'):
```

Comment:
yes, that's exactly what I meant. That way we will test that at least Trilinos interface to matrices/vectors works. :+1: